ydb_sqlalchemy/sqlalchemy/__init__.py

```diff
@@ -87,15 +87,30 @@ def visit_FLOAT(self, type_: sa.FLOAT, **kw):
     def visit_BOOLEAN(self, type_: sa.BOOLEAN, **kw):
```

Comment:
I have a doubt about  `visit_uuid` and implicit convert uuid to UTF8: YDB has own uuid type. Now it can't stored to a table, but it can be (and will sometime) in future. Then convert from uuid to UTF8 will see strange.

I suggest to not convert uuid to UTF-8 into SDK and suggest users to the conversion in customers code. It is simple as `str(var)`

Reply:
Discussed in offline: will solved in a separate PR